WorkloadManagementSystem/Utilities/RabbitMQSynchronizer.py

```diff
@@ -0,0 +1,103 @@
+'''RabbitMQSynchronizer
+
+  RabbitMQSynchronizer keeps the RabbitMQ user database  synchronized with the CS
+  RabbitMQ user database is updated according to changes in CS.
+'''
+
+from DIRAC import S_OK
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry import hostHasProperties
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getDNsInGroup
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getDNForHost
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getHosts
+from DIRAC.Core.Utilities.RabbitMQAdmin import getAllUsers, deleteUsers
+from DIRAC.Core.Utilities.RabbitMQAdmin import setUsersPermissions, addUsersWithoutPasswords
+from DIRAC.ResourceStatusSystem.Utilities import CSHelpers
+
+class RabbitMQSynchronizer(object):
+
+  def __init__( self ):
+
+    # Warm up local CS
+    # I am not sure whether it is needed but
+    # it was used in DIRAC.ResourceStatusSystem.Utilities.Synchronizer
+    CSHelpers.warmUp()
+    self._accessUserGroup = 'lhcb_pilot'  #only users belonging to group with this property are allowed to connect
```

Comment:
I added two functions getAllowedGroupName() getAllowedHostProperty() to separate it. 
It will be available in the next push